apps/drawio/26.0.0/docker-compose.yml

```diff
@@ -1,6 +1,6 @@
 services:
   drawio:
-    image: jgraph/drawio:26.0.0
+    image: jgraph/drawio:26.0.2
     container_name: ${CONTAINER_NAME}
     restart: unless-stopped
     networks:
```

Comment:
The code change appears to be an update from `image: jgraph/drawio:26.0.0` to `image: jgraph/drawio:26.0.2`. If you want feedback on any specific aspect, such as security updates, feature enhancements, or performance improvements, please let me know!